tools/SourceKit/lib/SwiftLang/SwiftEditorInterfaceGen.cpp

```diff
@@ -381,12 +381,16 @@ SwiftInterfaceGenContext::create(StringRef DocumentName,
   ASTContext &Ctx = CI.getASTContext();
   CloseClangModuleFiles scopedCloseFiles(*Ctx.getClangModuleLoader());
 
-  // Load standard library so that Clang importer can use it.
+  // Load standard library and implict imports so that Clang importer can use
+  // it.
   auto *Stdlib = Ctx.getModuleByIdentifier(Ctx.StdlibModuleName);
   if (!Stdlib) {
     ErrMsg = "Could not load the stdlib module";
     return nullptr;
   }
```

Comment:
Do we still need this? `CompilerInstance::setup` checks it up front if it is needed ("needed" being based on requested action and StdlibKind).

Reply:
Indeed. Removed it.